### PR TITLE
add: `xdg-terminal-exec-git`

### PR DESCRIPTION
**Pull request by Xdavius**

### CODE REVIEW

File: packages/xdg-terminal-exec-git/xdg-terminal-exec-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+pkgdesc="Proposed standard to launching desktop apps with Terminal=true"
+arch=("amd64")
+url='https://github.com/Vladimir-csp'
+makedepends=('git')
+source=("git+${url}/${gives}")
+sha256sums=('SKIP')
+maintainer=("xdavius <xdavius@github.com>")
+
+package() {
+  cd "${_archive}"
```
Comment (Elsie19) [inline packages/xdg-terminal-exec-git/xdg-terminal-exec-git.pacscript:13]: Do not use `${_archive}`.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/xdg-terminal-exec-git/xdg-terminal-exec-git.pacscript (hunk 1 above)
Comment (Xdavius) [reply packages/xdg-terminal-exec-git/xdg-terminal-exec-git.pacscript:13]: > Do not use `${_archive}`.

??

Comment (Xdavius) [reply packages/xdg-terminal-exec-git/xdg-terminal-exec-git.pacscript]: done

File: packages/xdg-terminal-exec-git/xdg-terminal-exec-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,16 @@
+pkgname="xdg-terminal-exec-git"
+gives="xdg-terminal-exec"
+pkgver="0.10.0"
+pkgdesc="Proposed standard to launching desktop apps with Terminal=true"
+arch=("amd64")
+url='https://github.com/Vladimir-csp'
```
Comment (Elsie19) [inline packages/xdg-terminal-exec-git/xdg-terminal-exec-git.pacscript:6]:
```suggestion
url='https://github.com/Vladimir-csp/xdg-terminal-exec'
```

File: packages/xdg-terminal-exec-git/xdg-terminal-exec-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+makedepends=('git')
+source=("git+${url}/${gives}")
```
Comment (Elsie19) [inline packages/xdg-terminal-exec-git/xdg-terminal-exec-git.pacscript:8]:
```suggestion
source=("git+https://github.com/Vladimir-csp/xdg-terminal-exec")
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**